### ISSUE TITLE
SileroTTS creates new audio file for each token

### ISSUE THREAD

**Issue by RJSprod** [oobabooga/text-generation-webui#461]
### Describe the bug

I've just performed a fresh install to confirm this.

Unless i turn on no stream, SileroTTS will attempt to create an audio file for each word / token. 

Silero should not attempt to create audio until the response is complete.

Silero extension output directory is being filled up with audio clips that only add one word to the previous file. Is this known to be broken like this?

Turning off stream works, but it means that the text stream doesn't work. Is there a way to turn off streaming for Silero only?

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

1. Enable Silero Extension
2. Disable Auto Play
3. Start Chat

### Logs

```shell
N/A
```

### System Info

```shell
Windows 11 / Firefox or Edge
```

**Comment by RJSprod** [comment]
Okay, so i followed this thread and manually merged the pull request changes into my Silero Extension and Modules Shared and Text Gen files --> https://github.com/oobabooga/text-generation-webui/pull/192

Not sure why this hasn't been placed in the main branch yet. This pull request is several days old and has merge approval. It works, but it's not in the main branch or updated via fresh install.

... 

I pulled the new silero extension python file as is

I added the one line of still_streaming to the shared file

And I inserted all of the still_streaming states (false / true) into the appropriate location of text generation file

...

Best wish this is just merged into the main branch correctly!
